content/blog/docker-essentials-for-front-end-developers/index.md

```diff
@@ -186,66 +205,96 @@ docker run $IMAGE_ID
+Instantiate the `image` into a `container` and replace the `CMD` in the _Dockerfile_ for
+an interactive shell you can play around with. This is particularly useful when your app
+fails to start, you can ssh into the container and manually start the app or make some debugging.
 
-
-// spawns a new container and replaces de entry point with bash and ssh into it, this
-is what you want for debugging
 ```bash
 docker run -ti $IMAGEID_OR_TAG bash
 ```
 
-Run a container with your local machine (host)'s app directory as a file system to
-edit your container's files in your host machine with your acustomed IDEs or text editors
+</br>
+
+Run a `container` that uses your host's (your machine) App directory and files.
+This allows you to use your regular development tools such as IDEs or Text Editors
+to edit your `container`'s files in place. This is particularly useful when debugging
+your App running inside a container.
+`$PWD` is your process working directory or the directory you are currently located in your shell instance,
+in this case we assume is your App's root directory.
+Your container `WORKDIR` should be the one you set in your _Dockerfile_.
+
 ```bash
 docker run -ti -v $PWD:your-container-workdir $IMAGEID_OR_TAG bash
 ```
 
-$PWD is your process workiding directory or the directory you are currently located in your shell instance,
-your container workdir should be the workdir your set in your Docekrfile (where all your app files will live)
+</br>
+
+SSH into an already running container.
+Useful for debugging a partially running `container` or `app` or
+for tweaking some details of your `container` or `app`.
+Most of the time I use the previous ones since I find them more useful.
 
-Ssh into an already running container, useful for debugging your Dockerfile or your app running inside a container
 ```bash
 docker exec -ti $CONTAINER_ID /bin/bash
 ```
 
+</br>
 
 ### Managing
 
-What are the current running containres?
+What are the current running `containers`?
+
 ```bash
 docker ps
 ```
 
-Kill a running container (grab the container id from the above output)
+</br>
+
+Kill a running `container` (get the `$CONTAINER_ID` from the above output).
+
 ```bash
 docker kill $CONTAINER_ID
 ```
 
-Check container logs
+</br>
+
+Check a `container`'s logs.
+
 ```bash
 docker logs $CONTAINER_ID
 ```
 
-Inspect a container (Docker debug information about volumnes, networks, ports, etc about a given container)
-(get container ip address)
+</br>
+
+Inspect a `container` (debug information about volumes, networks, ports, etc about a given container).
+In this case we get the `container`'s IP Address.
+
 ```bash
 docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $CONTAINER_ID
 ```
 
-Check the [docs][4] for a more complete coverage.
-
-
+</br>
 
+Check the [docs][4] for a more complete coverage.
 
+## Closing
 
+These are the basic knowledge I think every Front End developer should have as a minimum,
```

Comment:
_knowledge_ is uncountable and so it would be "this is the basic knowledge" or maybe it could be "these are the main basic pieces of knowledge" o algo así (?)